### PR TITLE
tikv(cdc): dynamic online change cdc configuration

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

add dynamic online change cdc configuration docs.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7241
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dynamic-config.md [hunk 1]
```diff
@@ -197,6 +197,11 @@ The following TiKV configuration items can be modified online:
 | `split.byte-threshold` | The threshold to execute `load-base-split` on a Region. If the traffic of read requests for a Region exceeds the `byte-threshold` for a consecutive period of time, this Region should be split. |
 | `split.split-balance-score` | The parameter of `load-base-split`, which ensures the load of the two split Regions is as balanced as possible. The smaller the value is, the more balanced the load is. But setting it too small might cause split failure. |
 | `split.split-contained-score` | The parameter of `load-base-split`. The smaller the value, the fewer cross-Region visits after Region split. |
+| cdc.min-ts-interval | The time interval at which Resolved TS is forwarded  |
+| cdc.old-value-cache-memory-quota | The upper limit of memory occupied by the TiCDC Old Value entries |
+| cdc.sink-memory-quota | The upper limit of memory occupied by TiCDC data change events |
+| cdc.incremental-scan-speed-limit | The upper speed limit of incremental scanning for historical data |
+| cdc.incremental-scan-concurrency | The maximum number of concurrent incremental scanning tasks for history data |
```
Comment (hfxsd) [inline dynamic-config.md:205]: Introduced from the following versions:
v5.1: v5.1.2
v5.0: v5.0.4
v4.0: v4.0.15

File: dynamic-config.md [hunk 2]
```diff
@@ -197,6 +197,11 @@ The following TiKV configuration items can be modified online:
...
```
Comment (TomShawn) [inline dynamic-config.md:204]:
```suggestion
| `cdc.incremental-scan-concurrency` | The maximum number of concurrent incremental scanning tasks for historical data |
```

File: dynamic-config.md [hunk 3]
```diff
@@ -197,6 +197,11 @@ The following TiKV configuration items can be modified online:
 | `split.byte-threshold` | The threshold to execute `load-base-split` on a Region. If the traffic of read requests for a Region exceeds the `byte-threshold` for a consecutive period of time, this Region should be split. |
 | `split.split-balance-score` | The parameter of `load-base-split`, which ensures the load of the two split Regions is as balanced as possible. The smaller the value is, the more balanced the load is. But setting it too small might cause split failure. |
 | `split.split-contained-score` | The parameter of `load-base-split`. The smaller the value, the fewer cross-Region visits after Region split. |
+| cdc.min-ts-interval | The time interval at which Resolved TS is forwarded  |
+| cdc.old-value-cache-memory-quota | The upper limit of memory occupied by the TiCDC Old Value entries |
+| cdc.sink-memory-quota | The upper limit of memory occupied by TiCDC data change events |
+| cdc.incremental-scan-speed-limit | The upper speed limit of incremental scanning for historical data |
```
Comment (TomShawn) [inline dynamic-config.md:203]:
```suggestion
| `cdc.min-ts-interval` | The time interval at which Resolved TS is forwarded  |
| `cdc.old-value-cache-memory-quota` | The upper limit of memory occupied by the TiCDC Old Value entries |
| `cdc.sink-memory-quota` | The upper limit of memory occupied by TiCDC data change events |
| `cdc.incremental-scan-speed-limit` | The upper limit on the speed of incremental scanning for historical data |
```

**Review by overvenus — APPROVED** [review]

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**